### ISSUE TITLE
nothing happen after run

### ISSUE THREAD

**Issue by gotodragon** [pocopico/tinycore-redpill#168]
run it on pve, nothing to see.
![nothing](https://user-images.githubusercontent.com/33269567/194718875-1d873a85-c461-43ec-beb6-4da6c8b5b662.jpg)

**Comment by pocopico** [comment]
Can you please run an rploader.sh update first?

**Comment by gotodragon** [comment]
yes, I can run rploader.sh, according to your guide step by step to do it, I found the same result.

**Comment by gotodragon** [comment]
So please see the course as list, help to find the problem,thanks a lot.

tc@box:~$ ./rploader.sh update
Checking Internet Access -> OK
Checking if a newer version exists on the main repo -> sha256sum: latestrploader.sh: No such file or directory
Version is current
tc@box:~$ ./rploader.sh fullupgrade
Performing a full TCRP upgrade
Warning some of your local files will be moved to /home/tc/old/xxxx.2022-Oct-09-09-05
Updating custom_config.json
Updating custom_config_jun.json
Updating global_config.json
Updating modules.alias.3.json.gz
mv: cannot stat 'old/modules.alias.3.json.gz.2022-Oct-09-09-05': No such file or directory
Updating modules.alias.4.json.gz
mv: cannot stat 'old/modules.alias.4.json.gz.2022-Oct-09-09-05': No such file or directory
Updating rpext-index.json
Updating user_config.json
Updating rploader.sh
chown: cannot access 'modules.alias.3.json.gz': No such file or directory
chown: cannot access 'modules.alias.4.json.gz': No such file or directory
gzip: modules.alias.*.gz: No such file or directory
Please make sure you are using the latest 1GB img before using backup option
Current /home/tc size is 16M , try to keep it less than 1GB as it might not fit into your image
Should i update the sdb with your current files [Yy/Nn]
Y^[^H^C
tc@box:~$ ./rploader.sh fullupgrade
Performing a full TCRP upgrade
Warning some of your local files will be moved to /home/tc/old/xxxx.2022-Oct-09-09-11
Updating custom_config.json
Updating custom_config_jun.json
Updating global_config.json
Updating modules.alias.3.json.gz
mv: cannot stat 'old/modules.alias.3.json.gz.2022-Oct-09-09-11': No such file or directory
Updating modules.alias.4.json.gz
mv: cannot stat 'old/modules.alias.4.json.gz.2022-Oct-09-09-11': No such file or directory
Updating rpext-index.json
Updating user_config.json
Updating rploader.sh
chown: cannot access 'modules.alias.3.json.gz': No such file or directory
chown: cannot access 'modules.alias.4.json.gz': No such file or directory
gzip: modules.alias.*.gz: No such file or directory
Please make sure you are using the latest 1GB img before using backup option
Current /home/tc size is 16M , try to keep it less than 1GB as it might not fit into your image
Should i update the sdb with your current files [Yy/Nn]
Y
Backing up home files to sdb : Backing up files to /mnt/sdb3//mydata.tgz  
Done.

tc@box:~$ ./rploader.sh serialgen DS3615xs
Serial Number for Model = 1330LW(I delete some)
Mac Address for Model DS3615xs = 00:11:32:D7(I delete some)
Should i update the user_config.json with these values ? [Yy/Nn]
Y
tc@box:~$ ./rploader.sh satamap
Machine is VIRTUAL Hypervisor=KVM
Found SCSI HBAs, We need to install the SCSI modules
scsi-5.10.3-tinycore64 is already installed!
Succesfully installed SCSI modules

Found "00:1f.2 Intel Corporation 82801IR/IO/IH (ICH9R/DO/DH) 6 port SATA Controller [AHCI mode] (rev 02)"
Detected 6 ports/0 drives. Mapping KVM q35 bogus controller after maxdisks

Found "06:07.0 Intel Corporation 82801IR/IO/IH (ICH9R/DO/DH) 6 port SATA Controller [AHCI mode] (rev 02)"
Detected 6 ports/1 drives. Mapping SATABOOT drive after maxdisks

Found SCSI/HBA "06:05.0 Red Hat, Inc Virtio SCSI" (1 drives)
lspci: -s: Invalid slot number
Found SCSI/HBA "" (0 drives)
lspci: -s: Invalid slot number
Found SCSI/HBA "" (0 drives)
lspci: -s: Invalid bus number
Found SCSI/HBA "" (0 drives)
lspci: -s: Invalid slot number
Found SCSI/HBA "" (0 drives)
lspci: -s: Invalid slot number
Found SCSI/HBA "" (0 drives)
lspci: -s: Invalid slot number
Found SCSI/HBA "" (0 drives)
lspci: -s: Invalid slot number
Found SCSI/HBA "" (0 drives)
lspci: -s: Invalid slot number
Found SCSI/HBA "" (0 drives)

Computed settings:
SataPortMap=11
DiskIdxMap=1010

Should i update the user_config.json with these values ? [Yy/Nn] Y
Done.
tc@box:~$ ./rploader.sh backup
Please make sure you are using the latest 1GB img before using backup option
Current /home/tc size is 16M , try to keep it less than 1GB as it might not fit into your image
Should i update the sdb with your current files [Yy/Nn]
Y
Backing up home files to sdb : Backing up files to /mnt/sdb3//mydata.tgz  
Done.

tc@box:~$ ./rploader.sh build bromolow-7.0.1-42218
bspatch does not exist, bringing over from repo
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (7) Failed to connect to raw.githubusercontent.com port 443: Connection refused
chmod: cannot access 'bspatch': No such file or directory
mv: cannot stat 'bspatch': No such file or directory
Rploader Version : 0.9.2.7
Loader source : https://github.com/pocopico/redpill-load.git Loader Branch : develop 
Redpill module source : https://github.com/pocopico/redpill-lkm.git : Redpill module branch : master 
Extensions :   redpill-misc 
Extensions URL : "https://github.com/pocopico/redpill-load/raw/develop/redpill-misc/rpext-index.json"
TOOLKIT_URL : https://sourceforge.net/projects/dsgpl/files/toolkit/DSM7.0/ds.bromolow-7.0.dev.txz/download
TOOLKIT_SHA : a5fbc3019ae8787988c2e64191549bfc665a5a9a4cdddb5ee44c10a48ff96cdd
SYNOKERNEL_URL : https://sourceforge.net/projects/dsgpl/files/Synology%20NAS%20GPL%20Source/25426branch/bromolow-source/linux-3.10.x.txz/download
SYNOKERNEL_SHA : 18aecead760526d652a731121d5b8eae5d6e45087efede0da057413af0b489ed
COMPILE_METHOD : toolkit_dev
TARGET_PLATFORM       : bromolow
TARGET_VERSION    : 7.0.1
TARGET_REVISION : 42218
REDPILL_LKM_MAKE_TARGET : dev-v7
KERNEL_MAJOR : 3
MODULE_ALIAS_FILE :  modules.alias.3.json
SYNOMODEL : ds3615xs_42218 
MODEL : DS3615xs 
Local Cache Folder : /mnt/sdb3/auxfiles
DATE Internet : 09102022 Local : 09102022
Checking Internet Access -> OK
Checking if a newer version exists on the main repo -> sha256sum: latestrploader.sh: No such file or directory
Version is current
Cloning into 'redpill-lkm'...
remote: Enumerating objects: 1398, done.
remote: Counting objects: 100% (470/470), done.
remote: Compressing objects: 100% (134/134), done.
remote: Total 1398 (delta 355), reused 405 (delta 323), pack-reused 928
Receiving objects: 100% (1398/1398), 2.78 MiB | 5.04 MiB/s, done.
Resolving deltas: 100% (918/918), done.
Cloning into 'redpill-load'...
remote: Enumerating objects: 2809, done.
remote: Counting objects: 100% (293/293), done.
remote: Compressing objects: 100% (162/162), done.
remote: Total 2809 (delta 148), reused 263 (delta 122), pack-reused 2516
Receiving objects: 100% (2809/2809), 118.30 MiB | 13.09 MiB/s, done.
Resolving deltas: 100% (1370/1370), done.
No extra build option or static specified, using default <static> 
Using static compiled redpill extension
Removing any old redpill.ko modules
Looking for redpill for : ds3615xs_42218 
Module does not contain platorm information for ds3615xs_42218
tc@box:~$

**Comment by pocopico** [comment]
Can you try another platform ?

**Comment by kpas8288** [comment]
> Checking if a newer version exists on the main repo -> sha256sum: latestrploader.sh: No such file or director

check your internet connection to see whether you can get this url?
`curl --location "https://raw.githubusercontent.com/pocopico/tinycore-redpill/master/rploader.sh"`

**Comment by gotodragon** [comment]
the redireciton by curl command like this:
 curl --location https://raw.githubusercontent.com/pocopico/tinycore-redpill/master/rploader.sh
curl: (7) Failed to connect to raw.githubusercontent.com port 443: Connection refused

**Comment by kpas8288** [comment]
> the redireciton by curl command like this: curl --location https://raw.githubusercontent.com/pocopico/tinycore-redpill/master/rploader.sh curl: (7) Failed to connect to raw.githubusercontent.com port 443: Connection refused

maybe you should pass the GFW
